pulp_2to3_migration/app/pre_migration.py

```diff
@@ -116,6 +118,14 @@
                 pulp2detail_pb.total -= 1
                 pulp2detail_pb.save()
                 continue
+        elif record._content_type_id in mutable_content_models.keys():
+            # This is a mutable content type, check if the migrated content is older
+            # than the incoming one. If suhc was found, detele it and create a new pulp2content
+            outdated = Pulp2Content.objects.filter(pulp2_last_updated__lt=record._last_updated,
```

Comment:
I think we don't expect more than one record here, so may it's arguably cleaner to use try/except with  Pulp2Content.objects.get, what do you think?
At least there is no need to use itemisation in this case.
Not critical, just caught my eye.

Reply:
I just realized that we already filter content in a way that only the updated one gets through, see line 83.
It means that there is no need to query anything here and at this stage we can just treat all the mutable content as outdated one. Do you agree or do I miss anything?

Reply:
we still need to query pulp2content table because what we work with in the for loop is the mongo unit collection. so record is the record from mongo, we need to find same in the pulp2content

Reply:
if you meant that we do not need to filter by pulp2_last_updated, but only by record.id, you are probably right, i just included last_updated to be super sure.